docs/VERSIONING.md

```diff
@@ -82,14 +83,17 @@ flowchart BT
 DiracX utilizes a structured approach to containerization:
 
 1. **Base Image**:
+
    - All container images start from `diracx/base`.
 
-2. **Specialized Base Images**:
+1. **Specialized Base Images**:
```

Comment:
also this number and the below are wrong

Reply:
dependabot.

I hate this. Why is it interfering with my PR ???